components/program/builds.tsx

```diff
@@ -1,81 +1,85 @@
 import { memo } from "react";
-import { Card, CardActionArea, Stack } from "@mui/material";
-import { styled } from "@mui/material/styles";
-import Typography from "@mui/material/Typography";
-import Box from "@mui/material/Box";
-import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+
+import { CalendarIcon } from "@heroicons/react/outline";
 import InputIcon from "@mui/icons-material/Input";
 import Link from "next/link";
 import Status from "../status";
 import FormatDate from "../../utils/format-date";
-
-const Item = styled(CardActionArea)(({ theme }) => ({
-  padding: theme.spacing(2),
-  color: theme.palette.text.secondary,
-  display: "flex",
-  justifyContent: "flex-start",
-}));
-
+const CardActionArea = ({ children }) => {
+  return (
+    <div className=" container m-3 mx-auto flex flex-col  justify-evenly gap-4 rounded-md p-2 align-baseline  text-[#666666] shadow-md sm:px-6 md:flex-row md:align-middle lg:px-8">
+      {children}
+    </div>
+  );
+};
+const Item = ({ children }) => {
+  return <CardActionArea>{children}</CardActionArea>;
+};
+const Box = ({ children }) => {
+  return <div className="mx-auto md:w-min">{children}</div>;
+};
+const Stack = ({ children, sx }) => {
+  return (
+    <div style={sx} className="overflow-hidden rounded-md">
+      <ul role="list" className="divide-y-2 divide-gray-200">
+        {children}
+      </ul>
+    </div>
+  );
+};
```

Comment:
Why move this outside the function? I don't think it is wrong, but it's not the same style the other components have, and it looks weird.